<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Postprozessor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="plainTextEdit_eldungen">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>551</width>
      <height>461</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>871</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_quelle">
       <property name="text">
        <string>Quellverzeichnis:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_quelle"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton__quelle">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_ziel">
       <property name="text">
        <string>Zielverzeichnis:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_ziel"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_ziel">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>191</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox_quelldat_erhalt">
       <property name="text">
        <string>Quelldateien erhalten</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_std_namen_zuweisen">
       <property name="text">
        <string>Standard-Dateinamen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_drehung">
       <property name="text">
        <string>Drehung Bauteile:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_drehung_0">
       <property name="text">
        <string>keine</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_drehung_90">
       <property name="text">
        <string>90 Grad</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_drehung_180">
       <property name="text">
        <string>180 Grad</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_drehung_270">
       <property name="text">
        <string>270 Grad</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_drehung_autom">
       <property name="text">
        <string>automatisch</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_dateien_auflisten">
       <property name="text">
        <string>Dateien auflisten</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_start">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>100</y>
      <width>111</width>
      <height>103</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_ausgbeformate">
       <property name="text">
        <string>Ausgabeformate:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_af_eigen">
       <property name="text">
        <string>eigenes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_af_ganx">
       <property name="text">
        <string>*.ganx</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_af_fmc">
       <property name="text">
        <string>*.fmc</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>896</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWerkzeug">
    <property name="title">
     <string>Werkzeug</string>
    </property>
    <addaction name="actionWerkzeug_ganx_anzeigen"/>
    <addaction name="actionWerkzeug_fmc_anzeigen"/>
   </widget>
   <widget class="QMenu" name="menuExtras">
    <property name="title">
     <string>Extras</string>
    </property>
    <addaction name="actionInfo"/>
    <addaction name="actionStandard_Namen_anzeigen"/>
   </widget>
   <addaction name="menuWerkzeug"/>
   <addaction name="menuExtras"/>
  </widget>
  <action name="actionWerkzeug_ganx_anzeigen">
   <property name="text">
    <string>Werkzeug ganx anzeigen</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionWerkzeug_fmc_anzeigen">
   <property name="text">
    <string>Werkzeug fmc anzeigen</string>
   </property>
  </action>
  <action name="actionStandard_Namen_anzeigen">
   <property name="text">
    <string>Standard-Namen anzeigen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
